torch/distributed/_shard/sharding_spec/chunk_sharding_spec_ops/matrix_ops.py

```diff
@@ -418,3 +418,8 @@ def sharded_layer_norm(args, kwargs, pg):
     extra_check=sharded_layer_norm_check,
     customized_func=sharded_layer_norm,
 )
+
+
+_register_sharded_op_on_local_tensor(
+    torch.nan_to_num,
+)
```

Comment:
Isn't this an elementwise op instead of a matrix_op?

Reply:
Right. Let me move it here.